Description: Upstream changes introduced in version 2.1.0-1~nattyubuntu4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 bumblebee (2.1.0-1~nattyubuntu4) natty; urgency=low
 .
   * Based on git-changes v. 1.6.67..
 .
 The person named in the Author field signed this changelog entry.
Author: Martin Juhl (MrMEEE) <mj@casalogic.dk>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- bumblebee-2.1.0.orig/install-files/bumblebee-configuration
+++ bumblebee-2.1.0/install-files/bumblebee-configuration
@@ -80,15 +80,13 @@ if [ -f /tmp/machineinfo ]; then
 fi
 
 if [ "$DISTRO" == "UBUNTU" ] ; then
-	rm -rf /etc/alternatives/gl_conf
-	ln -s /usr/lib/mesa/ld.so.conf /etc/alternatives/gl_conf
+	update-alternatives --remove gl_conf /usr/lib/nvidia-current/ld.so.conf
 	rm -rf /etc/alternatives/xorg_extra_modules
 	rm -rf /etc/alternatives/xorg_extra_modules-bumblebee
 	rm -rf /usr/lib/xorg/x11-extra-modules/xorg
 	rm -rf /usr/lib/xorg/extra-modules
 	rm -rf /usr/lib/nvidia-current/xorg/xorg
-	rm -rf /etc/alternatives/x86_64-linux-gnu_gl_conf
-	ln -s /usr/lib/mesa/ld.so.conf /etc/alternatives/x86_64-linux-gnu_gl_conf
+	update-alternatives --remove x86_64-linux-gnu_gl_conf /usr/lib/nvidia-current/ld.so.conf
 	ln -s /usr/lib/nvidia-current/xorg /etc/alternatives/xorg_extra_modules-bumblebee
 	ldconfig
 fi
